pytest_pycharm.py

```diff
@@ -28,5 +28,9 @@ def pytest_exception_interact(node, call, report):
             debugger = pydevd.get_global_debugger()
         pydevd_tracing.SetTrace(None)  # no tracing from here
         debugger.handle_post_mortem_stop(thread, frame, frames_by_id, exception)
+        try:
+            debugger.handle_post_mortem_stop(thread, frame, frames_by_id, exception)
+        except AttributeError:
```

Comment:
Maybe let's leave a comment saying that this is the way to go starting from pycharm 2019.2?

Reply:
Makes sense. Also, I'll change the order to try 2019.2 approach first and fallback to old one.